### PR TITLE
packaging monae 0.0.4

### PR DESCRIPTION
**Pull request by affeldt-aist**
compatibility with mathcomp 1.10.0 and minor improvements

### CODE REVIEW

File: released/packages/coq-monae/coq-monae.0.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+models, examples of monadic equational reasoning, and an application
+to program semantics.
+"""
+tags: [
+  "category:Computer Science/Semantics"
+  "keyword: monad"
+  "keyword: effect"
+  "keyword: probability"
+  "logpath:monae"
+  "date:2019-11-09"
```
Comment (palmskog) [inline released/packages/coq-monae/coq-monae.0.0.4/opam]:
```suggestion
  "date:2019-12-06"
```

File: released/packages/coq-monae/coq-monae.0.0.4/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "coq-infotheo" { >= "0.0.6" }
+]
+synopsis: "Monae"
+description: """
+This repository contains a formalization of monads including several
+models, examples of monadic equational reasoning, and an application
+to program semantics.
+"""
+tags: [
+  "category:Computer Science/Semantics"
```
Comment (palmskog) [inline released/packages/coq-monae/coq-monae.0.0.4/opam]:
```suggestion
  "category:Computer Science/Semantics and Compilation/Semantics"
```
To match the "official" category name.

File: released/packages/coq-monae/coq-monae.0.0.4/opam (hunk 1 above)
Comment (affeldt-aist) [reply released/packages/coq-monae/coq-monae.0.0.4/opam]: DONE

File: released/packages/coq-monae/coq-monae.0.0.4/opam (hunk 2 above)
Comment (affeldt-aist) [reply released/packages/coq-monae/coq-monae.0.0.4/opam]: DONE

**Comment by affeldt-aist** [comment]
Thank you for your check!

**Pull request merged**